app/upgrades/v1_6.go

```diff
@@ -204,7 +204,19 @@ func V010601UpgradeHandler(
 	appKeepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-		if isMainnet(ctx) {
+		if isMainnet(ctx) || isTest(ctx) {
```

Comment:
We should think about using early returns here instead of this huge nested block just to avoid repeating a return